src/main/java/org/mule/service/http/impl/service/server/grizzly/GrizzlyServerManager.java

```diff
@@ -139,15 +140,15 @@ private void configureServerSocketProperties(TCPNIOTransportBuilder transportBui
    * Starts the transport if not started. This is because it should be started lazily when the first server is registered
    * (otherwise there will be Grizzly threads even if there is no HTTP usage in any app).
    */
-  private void startTransportIfNotStarted() throws IOException {
+  private void startTransportIfNotStarted() throws ServerCreationException {
     withContextClassLoader(this.getClass().getClassLoader(), () -> {
       if (!transportStarted) {
         transportStarted = true;
         transport.start();
       }
       return null;
-    }, IOException.class, e -> {
-      throw new IOException(e);
+    }, ServerCreationException.class, e -> {
```

Comment:
No, the `transport.start()` bit could throw an `IOException`, in which case the handler is called to throw the `ServerCreationException` one instead. If a `ServerCreationException` was thrown it would be re thrown as is.